docs/iris/src/whatsnew/contributions_2.1/newfeature_2018-May-03_multidim_collapse.txt

```diff
@@ -0,0 +1,5 @@
+* The partial collapse of multi-dimensional auxiliary coordinates is now
+  supported. Collapsed bounds span the range of the collapsed dimension(s).
+  *Note* the collapsed points now represent the mean over the collapsed
```

Comment:
Heads up @SciTools/iris-devs. This is a big change, but IMO makes a lot of sense. If uses really care about the mid-point, it can be computed from the bounds...

Reply:
>  fixing up a bunch of tests that he could have got away with not having done

Apologies @duncanwp, I hadn't really spotted that aspect ...

Reply:
I could switch the tests back, but I'd like to keep the option of using the mean. Could we add a keyword to Coord.collapse? Or create a futures/config option?

Reply:
To be clear, the proposal is to create two PRs, one implementing the MEAN, one implementing the multi-dim collapse. I don't think there is any discussion remaining regarding the multi-dim collapse, but there is still a conversation to be had about MEAN being the correct approach.

Is that how you've understood it too @duncanwp?